crossing-training.lic

```diff
@@ -22,6 +22,7 @@ class CrossingTraining
     @running = true
     @idling = false
     @settings = get_settings(variable.drop(1) + %w(art athletics perform stealing))
+    @settings.outdoor_room ||= 674
```

Comment:
No idea what you think of adding an outdoor_room setting. I was using this while testing stuff in 705.